.github/workflows/ci-workflow.yml

```diff
@@ -868,7 +868,7 @@ jobs:
 
       - name: Extension test
         run: |
-          python3 scripts/generate-tinysnb.py
+          python3 scripts/generate-tinysnb.py relwithdebinfo
```

Comment:
I think you can make generate-tinysnb.py uses the relwithdebinfo binary by default.
It is not used anywhere else?

Reply:
changed